warehouse/malware/tasks.py

```diff
@@ -24,3 +27,60 @@ def run_check(task, request, check_name, obj_id):
     except Exception as exc:
         request.log.error("Error executing check %s: %s", check_name, str(exc))
         raise task.retry(exc=exc)
+
+
+@task(bind=True, ignore_result=True, acks_late=True)
+def sync_checks(task, request):
+    code_checks = inspect.getmembers(checks, inspect.isclass)
+    request.log.info("%d malware checks found in codebase." % len(code_checks))
+
+    all_checks = request.db.query(MalwareCheck).all()
+    active_checks = {}
+    wiped_out = {}
+    for check in all_checks:
+        if not check.is_stale:
+            if check.state == MalwareCheckState.wiped_out:
+                wiped_out[check.name] = check
+            else:
+                active_checks[check.name] = check
+
+    if len(active_checks) > len(code_checks):
+        request.log.error(
+            "Found %d active checks in the db, but only %d checks in \
+                code. Please manually move superfluous checks to the wiped_out state \
```

Comment:
+1 to logging and crashing out, It would be great to note which checks are being considered superfluous in the error message. could a comparison of the sets of `name` and/or `version` provide this?

Reply:
Sure!